lib/t/matches.ex

```diff
@@ -1254,7 +1254,7 @@ defmodule T.Matches do
     undying_events_q =
       MatchEvent
       |> where(match_id: parent_as(:match).id)
-      |> where([e], e.event == "call_start" or e.event == "meeting_report")
+      |> where([e], e.event == "call_start" or e.event == "contact_offer")
```

Comment:
turned out we already had `contact_offer` match_event
and since `meeting_report` can only be inserted after `contact_offer` => no need to insert `contact_offer` for every `meeting_report` in a migration

Reply:
Right, I ran

```sql
select mr.match_id from (select * from match_events where event = 'meeting_report') mr
left join (select * from match_events where event = 'contact_offer') co on mr.match_id = co.match_id
where co.match_id is null;
```

and it returns 0 results. It checks that every meeting report has a corresponding contact offer